### PR TITLE
Improve Oral-B documentation

### PR DESCRIPTION
**Pull request by mountainsandcode**
## Proposed change
Add a note about the fact that one of the sensors (and only one of them) requires an active bluetooth connection and close proximity. This is a frequent source of confusion for users, as seen in https://github.com/home-assistant/core/issues/90711

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [x] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/oralb.markdown [hunk 1]
```diff
@@ -48,3 +48,7 @@ The integration can discover most Bluetooth-enabled Oral-B toothbrushes. Brushes
 - Time - total brushing time in seconds.
 - Toothbrush state - whether the toothbrush is running, idle.
 - Battery - toothbrush battery percentage.
+
+<div class='note'>
+Updates of the battery sensor require an active bluetooth connection and relatively close proximity. If you are using a [bluetooth proxy](https://www.home-assistant.io/integrations/bluetooth/#remote-adapters-bluetooth-proxies), please ensure it supports active connections. All other sensors (i.e., all sensors except the battery sensor) will update with active or passive connections.
```
Comment (c0ffeeca7) [inline source/_integrations/oralb.markdown]:
```suggestion
Updates of the battery sensor require an active Bluetooth connection and relatively close proximity. If you use a [bluetooth proxy](https://www.home-assistant.io/integrations/bluetooth/#remote-adapters-bluetooth-proxies), please ensure it supports active connections. All the other sensors update with active or passive connections. 
```

**Review by c0ffeeca7 — APPROVED** [review]
Thank you, @mountainsandcode :+1:

File: source/_integrations/oralb.markdown [hunk 2]
```diff
@@ -48,3 +48,7 @@ The integration can discover most Bluetooth-enabled Oral-B toothbrushes. Brushes
 - Time - total brushing time in seconds.
 - Toothbrush state - whether the toothbrush is running, idle.
 - Battery - toothbrush battery percentage.
+
+<div class='note'>
+Updates of the battery sensor require an active Bluetooth connection and relatively close proximity. If you use a [bluetooth proxy](https://www.home-assistant.io/integrations/bluetooth/#remote-adapters-bluetooth-proxies), please ensure it supports active connections. All the other sensors update with active or passive connections.
```
Comment (bdraco) [inline source/_integrations/oralb.markdown:53]: This link does look like it formatted correctly in the preview

Comment (c0ffeeca7) [reply source/_integrations/oralb.markdown:53]: Right. Thank you for the catch. Let me fix that.

**Pull request merged**